build/common/installer/scripts/tomlparser.rb

```diff
@@ -150,6 +151,21 @@ def populateSettingValuesFromConfigMap(parsedConfig)
       ConfigParseErrorLogger.logError("Exception while reading config map settings for container log schema version - #{errorStr}, using defaults, please check config map for errors")
     end
 
+    # Get multiline log stitching setting
+    begin
+      if !parsedConfig[:log_collection_settings][:stitch_multiline_logs].nil? && !parsedConfig[:log_collection_settings][:stitch_multiline_logs][:enabled].nil?
+        @logStitchMultiline = parsedConfig[:log_collection_settings][:stitch_multiline_logs][:enabled]
+        puts "config::Using config map setting for multiline logging"
+
+        if @containerLogSchemaVersion.strip.casecmp("v2") !=0
```

Comment:
why are we overriding to v2? This will break existing customers who are using v1 when they enable multi-line feature. We should make the v2 is pre-req for the multi-line log feature. if the  customer applied multi-line log feature on v1 schema, then this setting should be ignored.